config/src/config/consensus_config.rs

```diff
@@ -133,8 +137,6 @@ impl Default for DelayedQcAggregatorConfig {
 /// and adjusts block sizes to "recalibrate it" to wanted range.
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct ExecutionBackpressureConfig {
-    /// At what latency does this backpressure level activate
```

Comment:
we want execution backpressure (i.e. recalibration) to apply without backlog, as long as execution blocks are large enough

Reply:
we activate backpressure if within last num_blocks_to_look_at=12 , there was at least min_blocks_to_activate=4  that lasted longer than min_block_time_ms_to_activate=100 , and estimated max block size based on that sample is smaller than currently used block size.
basically those 3 are enough to calibrate how long blocks take, and to estimate how long they should be cut at.
If we wait for back_pressure_pipeline_latency_limit_ms (was 500) to kick in then it only kicks in if there is large enough backlog (and then latency is already bad) - but this backpressure is best in avoiding the backlog to form in the first place. Tests showed that it kicks in , helps, then it stops activating because there is backlog up, then backlog increases , then it kicks in again, etc, causing latency to spike up, then down.
maybe “gas recalibration” is better term than “backpressure” for this.